### PR TITLE
CloudHub Mule Log Feature

### PR DESCRIPTION
**Pull request by hannanelson**
[View Log Data _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/6307105/View.Log.Data._.MuleSoft.Documentation.pdf)

### CODE REVIEW

**Review by pmincz — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 1]
```diff
@@ -9,10 +9,11 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
 
-CloudHub provides access to log data that includes deployment messages and events for each worker.
-
-Logs and diagnostics are currently available only for applications deployed to CloudHub workers. In cases when applications are deployed to a local server, you can send data to external analytics software.
+CloudHub provides access to log data that includes deployment messages and events for each worker and logs for Mule runtime engine.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: A little thought about avoiding "to log data that includes...logs for...":
```suggestion
CloudHub provides access to log data that includes worker deployment messages and events and Mule runtime engine logs.
```

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 2]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
-View logs for applications deployed to CloudHub.
+To view logs for applications deployed to CloudHub:
 
-. In Runtime Manager, click *Applications* in the left menu, and then click the application that is deployed to CloudHub to open the management panel on the right.
-. Click *Logs*.
+. In Anypoint Platform, select *Runtime Manager*.
+. In the Runtime Manager navigation menu, click *Applications*, and then click the application name.
+. In the navigation menu, click *Logs*.
++
+The console displays the application logs.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: Second sentence nec.?

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 3]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
 +
-.The arrow shows the checkbox to filter log output by deployment on the *Logs* page.
-image::cloudhub-logs-deploy.png[Filter log output by deployment on the Logs page]
+.The arrow shows the checkbox to filter log output by deployment in the *Deployments* section.
+image::cloudhub-logs-deploy.png[Filter log output by deployment in the Deployments section]
 
 == Download Logs
 
-Each worker in each CloudHub deployment has separate logs you can download.
+For each worker in a CloudHub deployment, you can download separate logs:
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: Here's an alternative for you to take or leave:
```suggestion
For each worker in a CloudHub deployment, you can download logs specific to that worker's activity:
```

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 4]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
+. In the *Deployments* section, select the date and worker for which you want to download logs.
+. Click the download icon for the worker, and select *Logs*:
 +
-.The arrow shows the icon to download log output by deployment on the *Logs* page.
-image::cloudhub-logs-download.png[Download log output by worker on the Logs page]
+.The arrow shows the icon to download log output by deployment in the *Deployments* section.
+image::cloudhub-logs-download.png[Download log output by worker in the Deployments section]
 
-The log file is a text file with the file extension `.log` in the format `<deployment-ID>-<instance-ID>.log`. You can view the log file with a text editor.
+CloudHub downloads the log output to a text file with the extension `.log` in the format `__deploymentID__.__instanceID__-__workerNumber__.log`.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: How about omitting repetition of the extension?
```suggestion
CloudHub downloads the log output to a text file in the format `__deploymentID__.__instanceID__-__workerNumber__.log`.
```

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 5]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
+You can view the log file with a text editor.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: Nonmodal-verb alternative:
```suggestion
Use a text editor to view the log file.
```

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 6]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
+. Click the download icon for the worker and select *Diagnostics*:
 +
-.The arrow shows the icon to download diagnostics by worker on the *Logs* page.
-image::cloudhub-logs-download-diag.png[Download diagnostics by worker on the Logs page]
+.The arrow shows the icon to download diagnostics by worker in the *Deployments* section.
+image::cloudhub-logs-diag-download.png[Download diagnostics by worker in the Deployments section]
 
-The downloaded log file is a text file with the `.txt` file extension and named in the format of `<app-name>-instance-<instance-ID>-<worker-number>-diag.txt`. You can view this file with a text editor.
+CloudHub downloads the diagnostic output to a text file with the extension `.txt` in the format `__appName__-instance-__instanceID__-__workerNumber__-diag.txt`. 
+You can view this file with a text editor.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: See edit comments at Lines 53-4.

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 7]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
 
 == Download Logs
 
-Each worker in each CloudHub deployment has separate logs you can download.
+For each worker in a CloudHub deployment, you can download separate logs:
 
-. In the *Deployment* filters section on the right, find the date and worker you want to view.
-. Click the download icon next to the worker, and select *Logs*:
+. In the *Deployments* section, select the date and worker for which you want to download logs.
+. Click the download icon for the worker, and select *Logs*:
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: See Line 92; comma not nec.:
```suggestion
. Click the download icon for the worker and select *Logs*:
```

File: modules/ROOT/pages/viewing-log-data.adoc [hunk 8]
```diff
@@ -23,63 +24,82 @@ See xref:monitoring::index.adoc[Anypoint Monitoring Overview].
...
+For example, the log records an entry when a Mule service or app starts, deploys, stops, or undeploys.
+
+ . In the *Deployments* section, select the date and worker for which you want to download Mule logs.
+ . Click the download icon for the worker and select *Mule Logs*:
++
+.The arrow shows the icon to download Mule log output by deployment in the *Deployments* section.
+image::cloudhub-logs-mule-download.png[Download Mule log output in the Deployments section]
+
+CloudHub downloads the Mule log output to a text file with the extension `.log` in the format `__appName__-instance-__instanceID__-__workerNumber__-mule_ee.log`.
+You can view the log file with a text editor.
```
Comment (kbostrom) [inline modules/ROOT/pages/viewing-log-data.adoc]: See edit comments at Lines 53-4.

**Review by kbostrom — DISMISSED** [review]
CE

**Review by pmincz — DISMISSED** [review]

**Review by pmincz — DISMISSED** [review]

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**